src/components/user-profile-info/UserProfileInfo.tsx

```diff
@@ -25,20 +33,30 @@ const UserProfileInfo: FC<UserProfileInfoProps> = ({
   lastName,
   date,
   reviewsCount,
-  sx = {}
+  sx = {},
+  _id,
+  role
 }) => {
   const { t } = useTranslation()
 
   const name = `${firstName} ${lastName}`
 
+  const userURL = createUrlPath(authRoutes.userProfile.path, _id, {
+    role
+  })
+
   return (
     <Box sx={spliceSx(styles.root, sx.root)}>
-      <Avatar
-        src={photo && `${import.meta.env.VITE_APP_IMG_USER_URL}${photo}`}
-        sx={spliceSx(styles.avatar, sx.avatar)}
-      />
+      <Link to={userURL}>
+        <Avatar
+          src={photo && `${import.meta.env.VITE_APP_IMG_USER_URL}${photo}`}
+          sx={spliceSx(styles.avatar, sx.avatar)}
+        />
+      </Link>
       <Box sx={spliceSx(styles.info, sx.info)}>
-        <Typography sx={spliceSx(styles.name, sx.name)}>{name}</Typography>
+        <Link style={{ textDecoration: 'none' }} to={userURL}>
```

Comment:
maybe it would be clearer to move this style to the file with styles of this component